src/main/java/web/pages/flipkart/HomePage.java

```diff
@@ -12,7 +13,7 @@
     public SelenideElement popUpCross = $x("//div[@class='mCRfo9']/div/div/button");
     public static SelenideElement setShoes = $x("//form/div/div/input");
     public static SelenideElement searchShoes = $x("//button[@class='vh79eN']");
-
+    public SelenideElement payment_page = $x("//a[contains(text(),'Payments')]");
```

Comment:
Change the name to something more meaningful like - paymentButton/paymentLink.
Also follow camel casing without '_'